ras_party/views/pending_survey_view.py

```diff
@@ -120,10 +120,13 @@ def post_pending_surveys():
     if not respondent:
         raise BadRequest('Originator unknown')
     batch_number = uuid.uuid4()
+    # logic to extract business list
+    business_id_list = [pending_surveys['business_id'] for pending_surveys in pending_surveys]
```

Comment:
same as my other comment, if you use set comprehension then you can drop the `set()` on the next line